scripts/setup_script_e2e.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up some stuff"
+
+# Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
+mkdir bin artifacts
+# Change the default location of go's bin directory (without affecting GOPATH). This is where compiled binaries will end up by default
+# for eg go get ginkgo later on will produce ginkgo binary in GOBIN
+export GOBIN="`pwd`/bin"
+export GOBIN_TEMP=$GOBIN
+# Set kubeconfig to current dir. This ensures no clashes with other test runs
+export KUBECONFIG="`pwd`/config"
+export ARTIFACTS_DIR="`pwd`/artifacts"
+export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
```

Comment:
`CUSTOM_HOMEDIR=$ARTIFACTS_DIR` should be `CUSTOM_HOMEDIR=$ARTIFACT_DIR` because for custom dir has the path set by prow team. I don't think exporting https://github.com/openshift/odo/pull/4460/files#diff-13486c16e4a0760249f14a4ebfc7f573868866f417260c72093497f868584487R21 makes sense in this case.

Reply:
I am not sure where are we using this `ARTIFACTS_DIR`

Reply:
It's kind of confusing because, ARTIFACT_DIR is set by the prow team, but we are using our PSI setup that may not work as intended on PRs.
I will remove the ARTIFACTS_DIR from the script for now and check if there is any error because of this env when triggered on PRs by prow after it gets merged . ARTIFACT_DIR is not preset if we are not using prow to trigger the test, so it will fail.
Does that sound fine for us?

Reply:
>  ARTIFACT_DIR is not preset if we are not using prow to trigger the test, so it will fail.

Yes.

Reply:
>  ARTIFACT_DIR is not preset if we are not using prow to trigger the test, so it will fail.

Yes.

Reply:
I think you can verify your current changes on jenkin, If I am not wrong.

Reply:
I think you can verify your current changes on jenkin, If I am not wrong.

Reply:
I guess we will need to add that env from jenkins side to test